### PR TITLE
Add protocol adapter options for tenant timeout and max payload

### PR DESCRIPTION
**Pull request by ctron**
### Type of change

- Enhancement / new feature

### Description

This primarily adds a way to set the "tenant timeout", and provides a default value. It also allows to set the max payload size.

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by ctron** [comment]
@enmasse-ci run tests profile=iot testcase=io.enmasse.systemtest.iot.shared.http.HttpAdapterTest

**Comment by ctron** [comment]
@enmasse-ci run tests profile=iot

File: pkg/controller/iotconfig/adapter.go [hunk 1]
```diff
@@ -203,6 +225,10 @@ func AppendHonoAdapterEnvs(config *iotv1alpha1.IoTConfig, container *corev1.Cont
 
 }
 
+func appendAdapterEnvVar(container *corev1.Container, a adapter, key string, value string) {
```
Comment (lulf) [inline pkg/controller/iotconfig/adapter.go]: Is it really appending or is applying more correct?

Comment (ctron) [reply pkg/controller/iotconfig/adapter.go]: Good question … I know the underlying method is named "apply or remove" … however the result will be that a new env-var entry will be added to the list of existing  env-vars.

Also is the statement contained in the method `AppendHonoAdapterEnvs` and the next calls which follows is named  `AppendTrustStores`.

**Pull request merged**